ursa_prod_attrib_value_multicompany/models/product_attribute.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 - TODAY, Ursa Information Systems
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class ProductAttributeValue(models.Model):
+    _inherit = "product.attribute.value"
```

Comment:
@smangukiya The company field should be on the `product.attribute.value.line`:
https://docs.google.com/document/d/1kqgUXW48JQ-IhmU-eACXn1IR8iRTF7aT0_Ys8zk43v8/edit

Reply:
@max3903 : please recheck them, I have updated changes. Thanks!